README.md

```diff
@@ -110,6 +112,40 @@ java -jar build/libs/naksha-2.0.6-all.jar custom-config
 
 Then use a web browser to connect to `localhost:8080`, an OK message should be displayed if the service is up and running.
 
+#### Running Naksha in container
+
+Naksha can be run in container as well. So far, only locally build image can be used.
+To get Naksha container running, one must do the following:
+
+1) Build the fat jar:
+     ```shell
+    ./gradlew shadowJar
+    ```
+2) Build the local image - you will need to define Naksha version that will be passed to the docker build.\
+   You can either do explicitly or utilize `gradle.properties` file.\
+   Explicit version:
+    ```shell
+   docker build -t local-naksha-app . --build-arg NAKSHA_VER=2.0.15 
```

Comment:
if we are able to avoid passing explicit version, then this command will need correction